Arrowgene.Ddon.Database/Files/Database/Script/schema_sqlite.sql

```diff
@@ -510,3 +510,18 @@ CREATE TABLE IF NOT EXISTS "ddon_reward_box" (
 	PRIMARY KEY("uniq_reward_id" AUTOINCREMENT),
 	CONSTRAINT "fk_ddon_reward_box_character_common_id" FOREIGN KEY("character_common_id") REFERENCES "ddon_character_common"("character_common_id") ON DELETE CASCADE
 );
+
+CREATE TABLE IF NOT EXISTS "ddon_quest_progress" (
+	"character_common_id"	INTEGER NOT NULL,
```

Comment:
When I asked last night, I was told pawns have some sort of quest progression. There are also a few commands related to it. I don't know the specifics, so if you can convince me otherwise I will change it. I actually asked for the feedback for this reason based on the previous code review where you made a similar comment :)

Reply:
I've just seen they were referring to pawn expeditions. I don't think these work anything like actual quests, if anything it's more similar to crafting, where pawns become unavailable for a certain time until it completes.

Reply:
Hmm, but I see there are quest commands like `IsOrderPawnQuest`.

Reply:
Wouldn't that be related with giving pawn orders? 🤔

Reply:
Order means to "order a quest" in these quest commands.

Reply:
Is this not the command you can get from your partner pawn then? Since they'll give you quests.
https://h1g.jp/dd-on/?%E3%83%91%E3%83%BC%E3%82%BD%E3%83%8A%E3%83%AB%E3%82%AF%E3%82%A8%E3%82%B9%E3%83%88#content_1_12

Reply:
So is this a problem which I need to change? I think it is probably fine and in the event we do find some need for the common id, we don't need to migrate a bunch of DBs to switch from character -> common id.